spec.yaml

```diff
@@ -88,6 +91,115 @@ paths:
       responses:
         '200':
           description: Successfully uploaded content
+
+  /repos/{owner}/{repo}/storage:
+    get:
+      tags:
+        - Storage Integrations
+      summary: List integrated storages in the repository
+      operationId: getBuckets
+      parameters:
+      - $ref: './components/parameters/Owner.yaml'
+      - $ref: './components/parameters/Repository.yaml'
+      responses:
+        '200':
+          description: Successfully listed integrated storages
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/StorageIntegration.yaml'
+              example:
+                [
+                  {
+                    "name": "bucket/prefix",
+                    "protocol": "s3",
+                    "list_path": "https://dagshub.com/Simon/baby-yoda-segmentation/dataset/storage/content/s3/bucket/prefix"
+                  }
+                ]
+
+  /repos/{owner}/{repo}/storage/content/{protocol}/{bucket}/{path}: 
+    get:
+      tags:
+        - Storage Integrations
+      summary: List contents in the path
+      operationId: getBucketContent
+      parameters:
+      - name: include_size
+        in: query
+        schema:
+          type: boolean
```

Comment:
`include_size` also appear on line 31. Would be nicer to extract it to a separate file to stay consistent :)